src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs

```diff
@@ -202,7 +202,7 @@ protected internal virtual void ReceiveByteArray(int id, byte[] data)
 
             if (id != ByteArraysToBeRevived.Count)
             {
-                throw new ArgumentOutOfRangeException($"Element id '{id}' cannot be added to the byte arrays to be revived with length '{ByteArraysToBeRevived.Count}'.");
+                throw new ArgumentOutOfRangeException($"Element id '{id}' cannot be added to the byte arrays to be revived with length '{ByteArraysToBeRevived.Count}'.", innerException: null);
```

Comment:
I don't think this is necessary? The other constructor already sets the inner exception to null. Am I missing something here?

Reply:
So there's a slight difference between the constructors of `ArgumentOutOfRangeException(string)` and this `ArgumentOutOfRangeException(string, Exception)`. Notably, if we use the former, the parameter is the `name` of the parameter that causes this exception, and such the error message will be something like "Argument 'Element id '{id}' cannot be added to the byte arrays to be revived with length' is out of range ...`. By using the 2 parameter constructor, we can specify we want the error message to be `Element id '{id}' cannot be added to the byte arrays to be revived with length...`.

![image](https://user-images.githubusercontent.com/14852843/120524896-69a21600-c38c-11eb-9064-a3f8c1c5d6ed.png)